server/service/src/backend_plugin/examples/amc/amc.js

```diff
@@ -36,6 +42,9 @@ let plugins = {
     const sql_result = sql(sql_statement);
     const response = {};
 
+    // Fill all item_ids with default
+    item_ids.forEach((itemId) => (response[itemId] = { average_monthly_consumption: 1 }));
```

Comment:
If there is not 'consumption' then it will be zero, changed to '1' to see effect of the plugin